packages/runtime/container-runtime/src/blobManager.ts

```diff
@@ -482,6 +493,28 @@ export class BlobManager extends TypedEventEmitter<IBlobManagerEvents> {
         entry.handleP.resolve(this.getBlobHandle(localId));
     }
 
+    private getTimeInfo(pendingEntry: PendingBlob, eventName: string, noTTLEventName: string) {
+        let timeLapseSinceLocalUpload: number = 0;
+        let expiredUsingLocalTime;
+        const minTTL: number  | undefined = pendingEntry.minTTL;
+        if(minTTL) {
```

Comment:
i would simply this an always just log all the data the is available, rather than changing the eventName base on the data avaialbe. its easy enough in telemetry tools to count row where a value is undefined, so i don't think the added bundle size from mutiple strings is worth it.

Reply:
You mean no adding any condition at all and just log every time we send the blob attach? There are 2 issues: 
1. useless logging will grow by a lot because there is no need to log on basically all scenarios but odsp (as far as I know is the only one sending TTL)
2.  data will have no sense doing operations with an undefined minTTL. 

If you believe logging all scenarios is worth it (maybe it is to still calculate time lapse) and undefined operations are not a big problem when analyzing the data then maybe it makes sense to remove the condition at all.

Maybe I got it wrong and you meant to say that the else condition is not needed and we should only use one eventName string

Reply:
i don't think there are that many blob uploads, and the existing code will produce a log line for every upload currently. so i don't quite understand point #1. if that is the case then you should only be logging when TTL exists. i believe that would also result in no data being logged, as nothing currently sets ttl

i believe both localUploadTime and serverUploadTime will be valid an useful without ttl, as it will give us some idea of how long it takes to upload in the worse case, which would be useful data to have